ppr-ui/src/interfaces/store-interfaces/state-interfaces/account-information-interface.ts

```diff
@@ -1,4 +1,4 @@
-// Account information interface
+import { AccountProductSubscriptionIF } from '@/interfaces'
```

Comment:
why this import?

Reply:
oops, I was going to put this in here and then decided to have it as it's own section in the store. Will remove